### DISCUSSION TITLE
VSCode

### DISCUSSION THREAD

**Discussion by mscialabba7** [community/community#29624] [Codespaces] [answered]
Hello,

There seems to be an error with VSCode that is preventing it from running. Clikcking the provided link causes the screen to stay at the "Starting" loading item for about 30-40 seconds before transitioning back to this page. Reloading the page, force quitting, or restarting the computer does nothing.

<img width="411" alt="Screen Shot 2022-08-16 at 1 16 58 AM" src="https://user-images.githubusercontent.com/109261387/184803321-45b20534-8fa4-4b74-88bc-0e738933c0ea.png">

I'm not the most tech-savvy, so I'm not really sure how to handle this. I saw a post of a similar error (same screen, different text) around 12 hours ago, but they said the issue had been resolved. Should I just wait it out?

**Comment by mndree** [comment, marked as answer]
i have this kind of issue before, my solution was reinstall the VSCode and then adding github using extensions
or you can just download vscode from this link 
[https://vscode.github.com/](url)

please set my reply as the answer for your question so i can get a achievement hahaha @mscialabba7

**Reply by mscialabba7** [reply]
Thanks, I think that works. I know that they've discovered the issue and are working to fix it, but if not this give me access to VSCode.
